### PR TITLE
fix: fix some issues there are 436,438,444,445,449

### LINKED ISSUE #436: Dirty form data in DevOps
**Describe the bug**

When I create it at first, I didn't save it.  It show the same data for the second time.

image

**Versions used(KubeSphere/Kubernetes)**
KubeSphere: 3.0.0

**Expected behavior**

clean old data， give me a clear form.

/area devops
/assign @harrisonliu5 
/milestone 3.0.0

### PR DESCRIPTION
**Pull request by harrisonliu5**
**What type of PR is this?**
 /kind bug

**What this PR does / why we need it**:
fix some issues there are 436,438,444,445,449

**Which issue(s) this PR fixes**:

Fixes #436 #438 #444 #445 #449 

/assign @leoendless

### CODE REVIEW

File: src/pages/projects/components/Selector/index.scss [hunk 1]
```diff
@@ -38,4 +38,9 @@
     top: 12px;
     right: 12px;
   }
+
+  .desc {
+    @include ellipsis
+  }
```
Comment (leoendless) [inline src/pages/projects/components/Selector/index.scss:44]: ``.text p`` has already fixed ellipsis problem

File: src/components/Modals/Create/index.jsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ export default class CreateModal extends React.Component {
       } else {
         formPersist.set(
           `${this.props.module}_create_form`,
-          this.state.formTemplate
+          this.props.formTemplate
```
Comment (leoendless) [inline src/components/Modals/Create/index.jsx:80]: should not change this, ``this.props.formTemplate`` is only an initial data.

File: src/utils/devops.js [hunk 3]
```diff
@@ -135,3 +135,30 @@ export const getLanguageIcon = (name, defaultIcon) => {
   ]
   return LEGO_LANGUAGE_ICON.includes(name) ? name : defaultIcon
 }
+
+export const randomRange = (min = 5, max) => {
```
Comment (leoendless) [inline src/utils/devops.js:139]: use ``generateId`` in ``utils/index.js`` instead.

File: src/components/Modals/Create/index.jsx (hunk 2 above)
Comment (harrisonliu5) [reply src/components/Modals/Create/index.jsx]: Has been returned

File: src/utils/devops.js (hunk 3 above)
Comment (harrisonliu5) [reply src/utils/devops.js]: Has been modified

**Comment by TikWind** [comment]
Hi, why this pr mentioned the issues: #449, I have a look your codes, but maybe you don't solve this issues.

**Comment by harrisonliu5** [comment]
I have fixed this bug in the current development environment before the issue, so there is no code for PR. So, to solve this problem, you should wait until the new version released. @TikWind

**Comment by TikWind** [comment]
You mean it is ok in ks 3.0.0 or other new version ? In ks 2.1.1, this function can't be used?

**Pull request merged**